seqr/views/apis/dataset_api.py

```diff
@@ -110,6 +112,37 @@ def add_variants_dataset_handler(request, project_guid):
     Family.bulk_update(
         request.user, {'analysis_status': Family.ANALYSIS_STATUS_ANALYSIS_IN_PROGRESS}, guid__in=family_guids_to_update)
 
+    if project.workspace_name and project.workspace_namespace:
+        if has_analyst_access(project):
+            safe_post_to_slack(
+                SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
+                """{num_sample} new samples are loaded in https://seqr.broadinstitute.org/project/{guid}/project_page
+                ```{samples}```
+                """.format(
+                    num_sample=len(matched_sample_id_to_sample_record),
+                    guid=project.guid,
+                    samples=[id for id in matched_sample_id_to_sample_record.keys()]
+                ))
+        else:
+            send_html_email(
+                """Hi {user},
+We are following up on your request to load data from AnVIL on {date}.
+We have loaded data from the AnVIL workspace “{namespace}/{name}” to the corresponding seqr project {proj_name}. {num_sample} samples are currently loaded. Let us know if you have any questions.
+Thanks,
+Data Manager from seqr
+""".format(
+                    user=request.user,
+                    date=project.created_date.date().strftime('%B %d, %Y'),
+                    namespace=project.workspace_namespace,
+                    name=project.workspace_name,
+                    proj_name=project.name,
+                    num_sample=len(matched_sample_id_to_sample_record),
+                ),
+                subject='AnVIL data have been loaded into seqr',
+                to=sorted([user['email'] for user in get_project_collaborators_by_username(request.user, project).values()
```

Comment:
you should just directly fetch the managers, rather than fetching the managers, fetching the collaborators, merging those lists together, and then separating them out. Theres also really no need to convert the collaborator list to json so you can then pull one field out

Reply:
Hmm, I should use a lower-level helper.